src/gui/internallinkwidget.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2022 by Claudio Cambra <claudio.cambra@nextcloud.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+#ifndef INTERNALLINKWIDGET_H
+#define INTERNALLINKWIDGET_H
+
+#include "QProgressIndicator.h"
+#include <QList>
+#include <QPushButton>
+
+#include "ui_internallinkwidget.h"
+
+namespace OCC {
+
+/**
+ * @brief The ShareDialog class
+ * @ingroup gui
+ */
+class InternalLinkWidget : public QWidget
+{
+    Q_OBJECT
+
+public:
+    explicit InternalLinkWidget(const QString &localPath,
+        QWidget *parent = nullptr);
+    ~InternalLinkWidget() = default;
```

Comment:
you are missing an `override` because you override the destructor of `QWidget` class

Reply:
funny thing is that adding this comment made me look at the drone CI report that is indeed broken and requests exactly the same [Drone CI check](https://drone.nextcloud.com/nextcloud/desktop/5231/2/5)

Reply:
Okay, added the `override` :)